singlestore-labs/demo-realtime-digital-marketing
# New Simulator button to toggle simulation status in database

## Summary

New Simulator toggle button in the nav bar that allows the user to strat or stop Martech dataset notifications and subscribers simulation.

## Test Plan

1. Pull the MR and from the terminal go to the web folder using the command `cd ./web`
2. In case you are using the RTDM project for the first time install dependencies using `yarn install`
3. Start the service using `yarn dev` and visit `http://localhost:3000`

You will see the Simulator toggle button In the Nav bar
![Screenshot_20230329_113254](https://user-images.githubusercontent.com/20814995/228440935-6d39b6be-15cc-49c1-a090-5d0730834414.png)


## Changes

- New Simulator button in Navbar tools that allows user to turn on/off simulation for martech

## Subscribers

@mivasconcelos  @TitoGrine
